packages/story-editor/src/components/checklist/checkpointContext/checkpointContext.js

```diff
@@ -118,10 +118,14 @@ function ChecklistCheckpointProvider({ children }) {
     dispatch(PPC_CHECKPOINT_ACTION.ON_PUBLISH_CLICKED);
   }, []);
 
+  const updateToAllowPriorityIssues = useCallback(
+    () => dispatch(PPC_CHECKPOINT_ACTION.ON_PUBLISH_CLICKED),
+    []
+  );
```

Comment:
💭 nit: maybe `showPriorityIssues` could work here too? I think `updateToAllow` is more verbose, so something simpler could be easier to read here. But this is fine too!

Reply:
that's a better name! thank you!